CMakeLists.txt

```diff
@@ -73,11 +73,7 @@ IF (APPLE)
 ENDIF()
 
 IF (UNIX)
-	IF (CMAKE_C_COMPILER_ID MATCHES "Clang")
-		SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -g -std=gnu99")
-	ELSE()
-		SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -g -std=gnu99 -Wno-deprecated-non-prototype")
-	ENDIF()
+	SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -g -std=gnu99 -Wno-deprecated-non-prototype")
```

Comment:
Hi David. I made this change as this was being generated by Clang for every file in the Github workflow:

```
warningwarning: unknown warning option '-Wno-deprecated-non-prototype' [-Wunknown-warning-option]
: unknown warning option '-Wno-deprecated-non-prototype' [-Wunknown-warning-option]
1 warning generated.
```

Would specifying the Clang version somewhere help?

Reply:
There is this [approach eventually ](https://cmake.org/cmake/help/v3.4/module/CheckCCompilerFlag.html). What do you think ?